docs/ruleset_functions/get_energy_required_to_heat_swh_use.md

```diff
@@ -0,0 +1,89 @@
+## get_energy_required_to_heat_swh_use
+
+Description: This function calculates the total energy required to heat the SWH use over the course of a year in btu.  Note - this function does not work for service water heating uses with use_units == "OTHER".  In this case, it will return 0.
+
+Inputs:
+- **swh_use**
+- **RMD**
+
+Returns:
+- **energy_required_by_space**: A dict where they keys are space_ids and values are the total energy required to heat the swh_use for that space.  If a swh_use is not assigned to any spaces, the key will be "NO_SPACES_ASSIGNED"
+
+Function Call:
+
+- get_spaces_served_by_SWH_use
+- get_obj_by_id
```

Comment:
I think get_obj_by_id is what I am used to seeing in RDSs but I see a lot of get_component_by_ID used in the other section 11 functions. Recommend consistency, or please correct me if they are different.